apps/freelance/src/pages/ProfileQuestions1/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Form, Input, Select } from 'antd';
+
+import { ProgressBar } from '@freelance/components';
+import { DefInput } from '@freelance/components';
+import { prBarStrColor, prBarTrailColor, prBarProfileQ1Per} from '@freelance/components'
+import { FormWrapper, Wrapper } from './styles';
+
+
+const ProfileQuestions1 = () => {
+  const { t } = useTranslation();
+  const { TextArea } = Input;
+  const { Option } = Select;
+
+  return (
+    <Wrapper>
+      <div>{t('description.profile p1 completion per')}</div>
+      <ProgressBar percent={prBarProfileQ1Per} strokeColor={prBarStrColor} trailColor={prBarTrailColor}/>
+      <FormWrapper>
+        <Form
+          name="profile_questions_1"
+          labelCol={{ span: 4 }}
+          wrapperCol={{ span: 14 }}
+          initialValues={{ remember: true }}
+          autoComplete="off"
+        > 
+          <Form.Item
+            label="Hourly Rate"
```

Comment:
text should be in text file

Reply:
-fixed (screenshot taken from translations/en.json)
![image](https://user-images.githubusercontent.com/95277626/200940193-5e0bc48d-a7dd-4f65-a6bc-9c5033bb0840.png)